python/run-tests

```diff
@@ -126,7 +126,20 @@ function run_streaming_tests() {
       KAFKA_ASSEMBLY_JAR="$f"
     done
 
-    export PYSPARK_SUBMIT_ARGS="--jars ${KAFKA_ASSEMBLY_JAR} pyspark-shell"
+    FLUME_ASSEMBLY_DIR="$FWDIR"/external/flume-assembly
+    JAR_PATH="${FLUME_ASSEMBLY_DIR}/target/scala-${SPARK_SCALA_VERSION}"
```

Comment:
Hi @zsxwing , this `JAR_PATH` will be different by default when you choose Maven against SBT as far as I know, so test with Maven build will fail to find the assembly jar, you could refer to #5632 .